contribute/developer-guide.md

```diff
@@ -5,7 +5,7 @@ This guide helps you get started developing KubeVela.
 ## Prerequisites
 
 1. Golang version 1.16+
-2. Kubernetes version v1.18+ with `~/.kube/config` configured.
+2. Kubernetes version v1.20+ with `~/.kube/config` configured.
```

Comment:
v1.18+ is okay. Developers needn't have a Kubernetes cluster v1.20+.

Reply:
Yes, but since we bump up to 1.20, we can not test what cases behave in v1.18. That may cause problem.

Reply:
Right now, we have to support v1.18, of which version is heavily used.